src/scripts.js

```diff
@@ -64,76 +71,87 @@ function getDate() {
   return date = dataDate;
 }
 
-function compareStepGoals() {
-  const avgGoal = allUsers.calculateAvgStepGoal();
-  const userGoal = user.dailyStepGoal;
-  stepGoalComparison.innerText = ` Your step goal is: ${userGoal}. Across all users the average step goal is: ${avgGoal}.`;
```

Comment:
If possible, could we shorten this sentence? For mobile styling it gets awkward and bulky.